app/views/includes/export-reductions.html

```diff
@@ -0,0 +1,3 @@
+{% if ((noAuth===true) or (userRole==='Data Admin') or (userRole==='Manager') or (userRole==='System Admin')) %}
+    <a href="/probation/{{ organisationLevel }}/{{ linkId }}/{{ screen }}/reductions-csv" class="button reduction-export" style="float: right; margin-right: 5px; margin-bottom: 10px">Export Reductions</a>
+{% endif %}
```

Comment:
`noAuth` has changed to `authorisation` and it should be compared against `false` value.